src/harness/testcases/configs/test_WINNF_FT_S_FAD_1/default.config

```diff
@@ -157,43 +127,80 @@
       },
       "creator": "admin_0",
       "ppaInfo": {
-        "ppaExpirationDate": "2019-01-30T14:55:51Z",
-        "ppaBeginDate": "2017-01-30T14:55:51Z",
+        "ppaRegionType": "URBAN",
+        "ppaExpirationDate": "2019-02-27T10:21:42Z",
+        "ppaBeginDate": "2017-02-27T10:21:42Z",
         "palId": [
-          "pal/01-2017/20041084200/1"
+          "pal/02-2017/20041084200/1"
         ]
       },
       "usage": "PPA",
-      "id": "zone/ppa/admin_0/pal/01-2017/20041084200/1/1cd5b1f9f0f14d63b627896ee47aae1d"
+      "id": "zone/ppa/admin_0/pal/02-2017/20041084200/1/4523f4e63b84459d8d413ab5d6f18c3c"
     }
   ],
   "grantRequests": [
     {
       "operationParam": {
         "maxEirp": 10,
         "operationFrequencyRange": {
-          "lowFrequency": 3620000000.0,
-          "highFrequency": 3630000000.0
+          "lowFrequency": 3620000000,
+          "highFrequency": 3630000000
         }
       }
     },
     {
       "operationParam": {
         "maxEirp": 10,
         "operationFrequencyRange": {
-          "lowFrequency": 3620000000.0,
-          "highFrequency": 3630000000.0
+          "lowFrequency": 3620000000,
+          "highFrequency": 3630000000
         }
       }
     },
     {
       "operationParam": {
         "maxEirp": 10,
         "operationFrequencyRange": {
-          "lowFrequency": 3620000000.0,
-          "highFrequency": 3630000000.0
+          "lowFrequency": 3620000000,
+          "highFrequency": 3630000000
         }
       }
     }
-  ]
+  ],
+  "palRecords ": [
+    [
+      {
+        "license": {
+          "licenseAreaExtent": "zone/census_tract/census/2010/20041084200",
+          "licenseFrequencyChannelId": "1",
+          "licenseId": "test_fcc_id_pal_0",
+          "licenseDate": "2017-02-27T10:21:42Z",
+          "licenseAreaIdentifier": "20041084200",
+          "licenseExpiration": "2019-02-27T10:21:42Z"
+        },
+        "registrationInformation": {
+          "callSign": "call_sign_pal_0",
+          "frn": "test_fcc_frn_pal_0",
+          "licenseConditions": "license_pal_0",
+          "registrationDate": "2017-02-27T10:21:42Z"
+        },
+        "userId": "test_user_id_a",
+        "licensee": "CBSD Operator No.1",
+        "channelAssignment": {
+          "primaryAssignment": {
+            "lowFrequency": 3550000000.0,
+            "highFrequency": 3560000000.0
+          }
+        },
+        "licenseStatus": "VALID",
+        "palId": "pal/02-2017/20041084200/1"
+      }
+    ]
+  ],
+  "sasHarness": {
+    "url": "https://localhost:9003/v1.2",
+    "serverCert": "F:\\GitHubTest\\Spectrum-Access-System\\src\\harness\\certs\\server.cert",
```

Comment:
This might be a good reason to use relative paths :) If I try to run default.config without modification, it will fail on my system. Manual modification is tiresome and automatic modification is risky.

Short answer: yes, let's please use relative paths.